dependency-injection/1.0/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Jakarta Depdency Injection 1.0"
+date: 2019-07-27T15:57:57-0700
```

Comment:
Just to follow @bshannon's lead, we probably don't need the time component for this timestamp.  Just the date should be sufficient.